src/main/scala/uk/gov/nationalarchives/consignmentexport/BagMetadata.scala

```diff
@@ -26,7 +26,11 @@ class BagMetadata(keycloakClient: KeycloakClient)(implicit val logger: SelfAware
   }
 
   private def getConsignmentDetails(consignment: GetConsignment, exportDatetime: ZonedDateTime): Map[String, Option[String]] = {
-    val seriesCode: Option[String] = consignment.series.map(_.code)
+    val seriesCode:Option[String]  = if(consignment.consignmentType.get.equals("judgment")){
```

Comment:
I would but the `consignmentType` val before this one. Then you don't have to call it twice, ie:
```
val consignmentType: Option[String] = consignment.consignmentType
    val seriesCode:Option[String]  = if (consignmentType.get.equals("judgment")) {
      Some("")
    } else {
      consignment.series.map(_.code)
    }
```
Just as a side note, I think it might be worthwhile in the future making the "consignment type" an actual type in the API, so don't have to use a string, as it needs to be used across multiple repos.